recipe/meta.yaml

```diff
@@ -10,7 +10,7 @@ source:
   md5: 552fcf215911b5bb7b7a2e5ad6a9a6e6
```

Comment:
You need to update the checksum. The latest version source this is `1fdef1d50270d22f8d89d9032fca2730`.